api/nakadi-event-bus-api.yaml

```diff
@@ -116,28 +116,6 @@ securityDefinitions:
         Grants access for consuming Event streams.
 
 paths:
-  /metrics:
```

Comment:
This path is available on the server but produces no data. It can be added back in when there's an implementation